.changeset/odd-clocks-fly.md

```diff
@@ -0,0 +1,9 @@
+---
+'@matrix-widget-toolkit/semantic-ui': patch
+'@matrix-widget-toolkit/testing': patch
+'@matrix-widget-toolkit/react': patch
+'@matrix-widget-toolkit/api': patch
+'@matrix-widget-toolkit/mui': patch
+---
+
+Bump `matrix-widget-api` from 1.1.1 to 1.3.1.
```

Comment:
why 1.3.1 not 1.2.1?

Reply:
It's the latest version of the https://github.com/matrix-org/matrix-widget-api/releases